doc/articles/guides/creating-custom-controls.md

```diff
@@ -0,0 +1,44 @@
+---
+
+uid: Uno.Tutorials.CreatingCustomControls  
+---  
+
+# Creating Custom Controls
+
+Custom controls in the Uno Platform enable developers to create reusable UI components that work seamlessly across iOS, Android, WebAssembly, and Windows. Uno Platform supports the same approach as WinUI/WinAppSDK and UWP, with a few specific considerations for cross-platform compatibility.
+
+> [!NOTE]  
+> For a more detailed explanation of creating custom controls, including dependency properties and control templates, refer to the official WinUI documentation: [Build XAML controls with C# | Microsoft Learn](https://learn.microsoft.com/en-us/windows/apps/winui/winui3/xaml-templated-controls-csharp-winui-3).
+
+If you're not familiar with UWP concepts such as **dependency properties**, **control templates**, and **visual states**, it might be useful to explore those topics before diving into custom controls. These are fundamental concepts for creating reusable, maintainable UI components:
```

Comment:
```suggestion
If you're not familiar with concepts such as **dependency properties**, **control templates**, and **visual states**, it might be useful to explore those topics before diving into custom controls. These are fundamental concepts for creating reusable, maintainable UI components:
```